core/embed/rust/src/protobuf/decode.rs

```diff
@@ -277,7 +282,7 @@ impl<'a> InputStream<'a> {
         let buf = self
             .buf
             .get(self.pos..self.pos + len)
-            .ok_or(Error::Missing)?;
+            .ok_or(error::end_of_buffer())?;
```

Comment:
`ok_or_else` would be better here, same in the bottom two cases.

Reply:
fixed in b406bd0a0ff986392c2fccef336a7b9201734023

i'd honestly expect the compiler to do this for me -- esp. in cases where it's `ok_or_else(|| something(foo.into()))`